Changelog.rst

```diff
@@ -44,69 +45,222 @@ Puppetserver auth.conf
 
 If you are upgrading from SIMP-6.0.0-0 to a later version:
 
-* The legacy auth.conf (``/etc/puppetlabs/puppet/auth.conf``) has been deprecated
-* ``pupmod-simp-pupmod`` will back up legacy puppet auth.conf after upgrade
+* The legacy ``auth.conf`` (``/etc/puppetlabs/puppet/auth.conf``) has been deprecated
+* ``pupmod-simp-pupmod`` will back up legacy puppet ``auth.conf`` after upgrade
 
-* The puppetserver's auth.conf is now managed by Puppet
-* You will need to re-produce any custom work done to legacy auth.conf in the
-  new auth.conf, via the ``puppet_authorization::rule`` defined type
+* The puppetserver's ``auth.conf`` is now managed by Puppet
+* You will need to re-produce any custom work done to legacy ``auth.conf`` in the
+  new ``auth.conf``, via the ``puppet_authorization::rule`` defined type
 * The stock rules are managed in ``pupmod::master::simp_auth``
 
-RPM Installation
-^^^^^^^^^^^^^^^^
+No Longer Delivering ClamAV DAT Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Given the wide spacing of SIMP releases, the team determined that it was
+effective for us to maintain the ``simp-rsync-clamav`` RPM with upstream ClamAV
+DAT file updates.
 
-FOSS SIMP
-"""""""""
+From this point forward, SIMP will not ship with updated ClamAV DAT files and
+we highly recommend updating your DAT files from the authoritative upstream
+sources.
 
-``sudo yum -y install simp-adapter simp``
+SNMP Support Added
 
-Puppet Enterprise SIMP
-""""""""""""""""""""""
+We have re-added SNMP support after a thorough re-assessment and update from
+our legacy ``snmp`` module. We now build upon a community module and wrap the
+SIMP-specific components on top of it.
+
+Preparing for Puppet 5
+^^^^^^^^^^^^^^^^^^^^^^
+
+We have been updating all of our modules to include tests for Puppet 5 and, so
+far, things have gone quite well. Many module version bumps were simply due to
+the fact that Puppet 5 tests were added and no functionality was changed.
```

Comment:
Not really true.  Some module bumps were for non-breaking dependency version bumps, like concat 3.0.0